core/storage/job/gae_models.py

```diff
@@ -36,11 +36,40 @@
 
 
 class JobModel(base_models.BaseModel):
-    """Class representing a datastore entity for a long-running job."""
+    """Class representing a datastore entity for a long-running job.
+
+    Attributes:
+        job_type: The job type.
+        time_queued_msec: The time at which the job was queued, in milliseconds since the epoch.
+        time_started_msec: The time at which the job was started, in milliseconds since the epoch.
+            This is never set if the job was canceled before it was started.
+        time_finished_msec: The time at which the job was completed, failed or canceled, in
+            milliseconds since the epoch.
+        status_code: The current status code for the job.
+        metadata: Any metadata for the job, such as the root pipeline id for mapreduce
+            jobs.
+        output: The output of the job. This is only populated if the job has status code
+            STATUS_CODE_COMPLETED, and is None otherwise.
+        error: The error message, if applicable. Only populated if the job has status
+            code STATUS_CODE_FAILED or STATUS_CODE_CANCELED; None otherwise.
+        has_been_cleaned_up:  Whether the datastore models associated with this job have been cleaned
+            up (i.e., deleted).
+        additional_job_params: Store additional params passed with job.
+    """
 
     @classmethod
     def get_new_id(cls, entity_name):
-        """Overwrites superclass method."""
+        """Overwrites superclass method.
+
+        This method is used to generate a new id for the job.
+
+        Args:
+            entity_name: a name for the running job.
```

Comment:
entity_name: str. The name of the job class associated with the job.

(Did you grep to see where JobClass.get_new_id() is called?)